src/utils.py

```diff
@@ -0,0 +1,50 @@
+import datetime
+import logging
+
+import pandas as pd
+import xarray as xr
+from ochanticipy import ChirpsDaily, CodAB, GeoBoundingBox
+
+from src import constants
+from src.datasource_extensions import FloodScan
+
+logger = logging.getLogger(__name__)
+
+
+def load_codab(admin_level=0):
+    codab = CodAB(constants.country_config)
+    # codab.download()
+    return codab.load(admin_level=admin_level)
+
+
+# question: should this preprocessing go here or in datasource_extensions?
```

Comment:
no idea tbh.  I'm probably the only one at this point, but i'd find a little mini session on how we use `datasouce_extensions` and even `ocha-anticipy` super helpful

Reply:
I think this is an old question from Tinka that I forgot to take out, I believe I fixed it in the next branch. But happy to show you more about how the DataSources class works!